packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -127,12 +127,25 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.updateData(x)}"
         >
-            <nimble-table-column-text field-name="firstName" placeholder="no value" column-id="first-name-column">
+            <nimble-table-column-text field-name="firstName" placeholder="no value" column-id="first-name-column" action-menu-slot="name-menu" action-menu-label="Configure name">
                 <nimble-icon-user title="First Name"></nimble-icon-user>
             </nimble-table-column-text>
-            <nimble-table-column-text field-name="lastName" placeholder="no value" column-id="last-name-column">Last Name</nimble-table-column-text>
+            <nimble-table-column-text field-name="lastName" placeholder="no value" column-id="last-name-column" action-menu-slot="name-menu" action-menu-label="Configure name">Last Name</nimble-table-column-text>
             <nimble-table-column-text field-name="favoriteColor" placeholder="no value" column-id="favorite-color-column">Favorite Color</nimble-table-column-text>
-            <nimble-table-column-text field-name="quote" placeholder="no value" column-id="quote-column">Quote</nimble-table-column-text>
+            <nimble-table-column-text field-name="quote" placeholder="no value" column-id="quote-column" action-menu-slot="quote-menu" action-menu-label="Configure quote">Quote</nimble-table-column-text>
+
+            <nimble-menu slot="name-menu">
```

Comment:
It would be good to have more explicit storybook documentation about how to configure an action menu (e.g. a storybook input that describes the slot name, what goes in it, and when you need to specify a column-id). This could happen in this PR or be tracked in one of the table column documentation follow up issues.

Reply:
I've updated our existing issue for column documentation to include the need to document action menus. (#1044)